OrbitLinuxTracing/LibunwindstackUnwinder.cpp

```diff
@@ -51,9 +51,6 @@ std::vector<unwindstack::FrameData> LibunwindstackUnwinder::Unwind(
   // those callstacks.
   if (unwinder.LastErrorCode() != 0 &&
       unwinder.frames().back().map_name != "[uprobes]") {
-    ERROR("%s at %#016lx",
```

Comment:
Would there be value in keeping this in the debug case only for debugging purposes?

Reply:
I'm not too sure but I can do that for now, it's always easy to remove.